src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write your code here
+const countryList = document.querySelectorAll('.population');
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+
+const total = [...countryList].reduce((acc, country) => {
+  const populationNumber = +country.innerHTML.split(',').join('');
+
+  return acc + populationNumber;
+}, 0);
+
+const average = total / countryList.length;
```

Comment:
If `countryList` is empty, dividing by zero will result in `Infinity`. Consider adding a check to ensure `countryList.length` is not zero before performing the division.